BloomGameStudio/BloomDiscordBot
# Dynamic proposal param fix

**Whats New**
- Updated config.ini settings to be capitalized to align with config.py 
- Added admin field, cleared members (this can be used in the future if desired)
- Noticed an error introduced with URL parsing post env switching update, amended
- updated modify_space settings to use env vars

**How to test**
- Launch the bot in Dev
- Create a proposal and publish it
- react to vote and observe URL posted
- Check logs to verify snapshot space modification & proposal submission
